config/routes.rb

```diff
@@ -1,7 +1,17 @@
 Rails.application.routes.draw do
-  root "auth#signup"
-
   post "/sessions", to: "auth#login"
   post "/users", to: "auth#signup"
   delete "/sessions", to: "auth#logout"
+
+  post 'hideout/create'
+  delete 'hideout/destroy'
+  put 'hideout/rename', to: "hideout#rename"
+  post "/hideout/add", to: "hideout#add"
+  post "/hideout/leave", to: "hideout#leave"
+  delete "/hideout/destroy", to: "hideout#destroy"
+
```

Comment:
We should rename these routes to follow best practices:

1. POST `/hideouts` to create a new hideout
2. DELETE `/hideouts` to delete a hideout
3. PUT `/hideouts/users` to add user to a hideout
4. DELETE `/hideouts/users` to leave a hideout